src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,47 @@
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import cn from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('century');
+  const centuryFilters = ['16', '17', '18', '19', '20'];
+  const sexFilters = [
```

Comment:
it's better to outside static variables outside of the component

fix all similar cases